com.unity.render-pipelines.universal/Shaders/SpatialMappingWireframe.shader

```diff
@@ -0,0 +1,153 @@
+Shader "Universal Render Pipeline/SpatialMappingWireframe"
+{
+    Properties
+    {
+        _WireThickness ("Wire Thickness", RANGE(0, 800)) = 100
+    }
+    SubShader
+    {
+        Tags {"RenderType" = "Opaque" "IgnoreProjector" = "False" "RenderPipeline" = "UniversalPipeline" }
+        LOD 100
+
+        Pass
+        {
```

Comment:
missing the note about where the code comes from, would be ideal to add this back in.
```
// Wireframe shader based on the the following
// http://developer.download.nvidia.com/SDK/10/direct3d/Source/SolidWireframe/Doc/SolidWireframe.pdf
```